flow/project.py

```diff
@@ -955,7 +1012,12 @@ def _create_submission_job_operation(self, entrypoint, default_directives, job,
         :rtype:
             :py:class:`SubmissionJobOperation`
         """
-        uneval_cmd = functools.partial(self._submit_cmd, entrypoint=entrypoint, job=job,
+        if isinstance(jobs, signac.contrib.job.Job):
+            jobs = [jobs]
```

Comment:
As this is an internal function, we should never have this branch of the conditional be true, if we do that is an inconsistency in the internals.

Reply:
This means then I'd have to edit the tests.